### ISSUE TITLE
Got exception while receiving SQS messages

### ISSUE THREAD

**Issue by Gr1N** [aio-libs/aiobotocore#124]
Hi, as I see `yarl` deps updated and I got to many exception...

```
Traceback (most recent call last):
  File "/Users/ng/Work/Projects/Python/packages/mspy/ms/core/transports/sqs.py", line 97, in _listen
    QueueUrl=queue_url, **receive_options)
  File "/Users/ng/.virtualenvs/venv42/lib/python3.6/site-packages/aiobotocore/client.py", line 78, in _make_api_call
    operation_model, request_dict)
  File "/Users/ng/.virtualenvs/venv42/lib/python3.6/site-packages/aiobotocore/endpoint.py", line 211, in _send_request
    exception)):
  File "/Users/ng/.virtualenvs/venv42/lib/python3.6/site-packages/aiobotocore/endpoint.py", line 243, in _needs_retry
    caught_exception=caught_exception, request_dict=request_dict)
  File "/Users/ng/.virtualenvs/venv42/lib/python3.6/site-packages/botocore/hooks.py", line 227, in emit
    return self._emit(event_name, kwargs)
  File "/Users/ng/.virtualenvs/venv42/lib/python3.6/site-packages/botocore/hooks.py", line 210, in _emit
    response = handler(**kwargs)
  File "/Users/ng/.virtualenvs/venv42/lib/python3.6/site-packages/botocore/retryhandler.py", line 183, in __call__
    if self._checker(attempts, response, caught_exception):
  File "/Users/ng/.virtualenvs/venv42/lib/python3.6/site-packages/botocore/retryhandler.py", line 251, in __call__
    caught_exception)
  File "/Users/ng/.virtualenvs/venv42/lib/python3.6/site-packages/botocore/retryhandler.py", line 269, in _should_retry
    return self._checker(attempt_number, response, caught_exception)
  File "/Users/ng/.virtualenvs/venv42/lib/python3.6/site-packages/botocore/retryhandler.py", line 317, in __call__
    caught_exception)
  File "/Users/ng/.virtualenvs/venv42/lib/python3.6/site-packages/botocore/retryhandler.py", line 223, in __call__
    attempt_number, caught_exception)
  File "/Users/ng/.virtualenvs/venv42/lib/python3.6/site-packages/botocore/retryhandler.py", line 359, in _check_caught_exception
    raise caught_exception
  File "/Users/ng/.virtualenvs/venv42/lib/python3.6/site-packages/aiobotocore/endpoint.py", line 269, in _get_response
    request.method, request.url, request.headers, request.body)
  File "/Users/ng/.virtualenvs/venv42/lib/python3.6/site-packages/aiobotocore/endpoint.py", line 200, in _request
    allow_redirects=False)
  File "/Users/ng/.virtualenvs/venv42/lib/python3.6/site-packages/aiohttp/client.py", line 577, in __iter__
    resp = yield from self._coro
  File "/Users/ng/.virtualenvs/venv42/lib/python3.6/site-packages/aiohttp/client.py", line 213, in _request
    proxy=proxy, proxy_auth=proxy_auth, timer=timer)
  File "/Users/ng/.virtualenvs/venv42/lib/python3.6/site-packages/aiohttp/client_reqrep.py", line 73, in __init__
    assert isinstance(url, URL), url
AssertionError: http://localhost:9324/
```

Any ideas?

**Comment by thehesiod** [comment]
can you give a testcase?  what version of aiohttp/botocore are you using?

**Comment by Gr1N** [comment]
Found issue, fixed with 0.9.6 release of `yarl` (https://github.com/aio-libs/yarl/releases/tag/0.9.6)...
